### PR TITLE
{lang}[intel/2016b] R v3.3.3

### PR DESCRIPTION
**Pull request by vanzod**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
cermis - Linux debian 9.3, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/c9d3451cd6f1af162256ed642de66deb for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2036.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/8a39449d5c3b054ac3f13a11a0273190 for a full test report.

File: easybuild/easyconfigs/r/R/R-3.3.3-intel-2016b-X11-20160819.eb [hunk 1]
```diff
@@ -0,0 +1,2250 @@
...
+description = """R is a free software environment for statistical computing and graphics."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+source_urls = ['http://cran.us.r-project.org/src/base/R-%(version_major)s']
+sources = [SOURCE_TAR_GZ]
+checksums = ['5ab768053a275084618fb669b4fbaadcc39158998a87e8465323829590bcfc6c']
+
+preconfigopts = 'BLAS_LIBS="$LIBBLAS" LAPACK_LIBS="$LIBLAPACK"'
+configopts = "--with-lapack --with-blas --with-pic --enable-threads --with-x=yes --enable-R-shlib"
```
Comment (boegel) [inline easybuild/easyconfigs/r/R/R-3.3.3-intel-2016b-X11-20160819.eb]: several of this stuff doesn't need to be here anymore, thanks to updates to the R easyblock, please see current (recent) R easyconfigs in `develop`

File: easybuild/easyconfigs/r/R/R-3.3.3-intel-2016b-X11-20160819.eb [hunk 2]
```diff
@@ -0,0 +1,2250 @@
...
+# Actually use Tcl/Tk
+configopts += ' --with-tcl-config=$EBROOTTCL/lib/tclConfig.sh --with-tk-config=$EBROOTTK/lib/tkConfig.sh '
+# Enable graphics capabilities for plotting.
+configopts += " --with-cairo --with-libpng --with-jpeglib --with-libtiff"
```
Comment (boegel) [inline easybuild/easyconfigs/r/R/R-3.3.3-intel-2016b-X11-20160819.eb]: these two `configopts +=` lines can be dropped imho

File: easybuild/easyconfigs/r/R/R-3.3.3-intel-2016b-X11-20160819.eb [hunk 3]
```diff
@@ -0,0 +1,2250 @@
...
+    'methods',
+    'splines',
+    'stats',
+    'stats4',
+    'tools',
+    'utils',
+    ('Rmpi', '0.6-6', {
+        'patches': ['Rmpi-0.6-5_impi5.patch'],
+        'source_tmpl': name_tmpl,
+        'source_urls': ext_urls,
```
Comment (boegel) [inline easybuild/easyconfigs/r/R/R-3.3.3-intel-2016b-X11-20160819.eb]: please use `exts_default_options` like we do in recent R easyconfigs, this is too much copy-pasta...

**Comment by boegel** [comment]
@vanzod ping on resolving the conflicts?

**Comment by boegel** [comment]
@vanzod re-ping?

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2115.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/794103524de1651b752af1f35b147a8e for a full test report.

**Comment by boegel** [comment]
@vanzod checksum problem for `Rmpi` (and probably also others), maybe re-run `--inject-checksums --force-download --sourcepath /tmp/pr5773` on this easyconfig to fix all checksums?

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 28 out of 59 (1 easyconfigs in this PR)
node2406.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/156be4ddf6c419b32646af810fc310d3 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2004.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/9be6029e6e14237b2cb2e3edb57ff707 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 31 out of 31 (1 easyconfigs in this PR)
node2594.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/d800001ba0234c84bd228a031e184e05 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @vanzod!

**Pull request merged**